hydra-node/test/Hydra/Ledger/CardanoSpec.hs

```diff
@@ -93,21 +91,23 @@ roundtripCBOR a =
 
 appliesValidTransaction :: Property
 appliesValidTransaction =
-  forAll genUTxO $ \utxo ->
-    forAllShrink (genSequenceOfValidTransactions defaultGlobals defaultLedgerEnv utxo) shrink $ \txs ->
-      let result = applyTransactions (cardanoLedger defaultGlobals defaultLedgerEnv) utxo txs
-       in isRight result
-            & counterexample ("Error: " <> show result)
-            & counterexample ("JSON for txs: " <> unpack (decodeUtf8With lenientDecode $ toStrict $ Aeson.encode txs))
-            & counterexample ("JSON for utxo: " <> unpack (decodeUtf8With lenientDecode $ toStrict $ Aeson.encode utxo))
+  forAllBlind (genSequenceOfValidTransactions defaultGlobals defaultLedgerEnv) $ \(utxo, txs) ->
```

Comment:
The disadvantage is that now we don't have any shrinking. I assume that by default shrinking `(a,b)` will result in shrinking independently `a` and `b` which is probably not what we want. This _might_ not be a problem for this particular property, but perhaps adding a note or `TODO` at the implementation site to later write a shrinker would be helpful?

Reply:
Hm.. maybe `(second shrink)` is already the shrinker we would want here?

Reply:
Note that we also have: 

```
-- | Like 'shrinkList', but more aggressive :)
--
-- Useful for shrinking large nested Map or Lists where the shrinker "don't have
-- time" to go through many cases.
shrinkListAggressively :: [a] -> [[a]]
```

in our Prelude :nerd_face: